apis/r/DESCRIPTION

```diff
@@ -34,7 +34,7 @@ Imports:
     Matrix,
     stats,
     bit64,
-    tiledb (>= 0.25.0),
+    tiledb (>= 0.25.0), tiledb (< 0.26.0),
```

Comment:
Conceptually this is correct but it has two issue.  First off, R only allows `<=`, not `<`.  Second, the way the releases run in the tiledb-r an earlier 'x' in 0.25.0.6 brought in 2.22.0-rc0.  So I have used `(<= 0.25.0.3)` with luck, looking at the commits `(<= 0.25.0.5)` should also work.  This should make little difference in practice.

```suggestion
    tiledb (>= 0.25.0), tiledb (<= 0.25.0.5)
```